style-index2.css

```diff
@@ -0,0 +1,379 @@
+@font-face {
+  font-family: 'UTM Avo';
+  font-style: normal;
+  font-weight: normal;
+  src:  url(./font/utm-avo.woff), format(woff);
+}
+@font-face {
+  font-family: 'UTM Avobold';
+  font-style: normal;
+  font-weight: normal;
+  src:  url(./font/utm-avobold.woff), format(woff);
+}
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  padding-left: 11.25%;
+  padding-right: 11.25%;
+}
+
+.img-css {
+  width: 100%;
+}
+
+/*Header*/
+header {
+  width: 100%;
+}
+.header-top {
+  display: flex;
+  margin-top: -8px;
+  justify-content: space-between;
+  height: 103px;
+  border-bottom: 1px solid #e9e9e9;
+}
+
+.header-top div {

```

Comment:
Are you sure, every single `div` inside `.header-top` will have display flex with center align-items? 

Consider using a class for this one.